lib/KeyboardAwareHOC.js

```diff
@@ -362,8 +362,8 @@ function listenToKeyboardEvents(ScrollableComponent: React$Component) {
 
     _resetKeyboardSpace = () => {
       const keyboardSpace: number = this.props.viewIsInsideTabBar
-        ? _KAM_DEFAULT_TAB_BAR_HEIGHT + this.props.extraScrollHeight || 0
```

Comment:
Have you tried to set `extraScrollHeight` to `0`?

Reply:
In fact, if you don't set `extraScrollHeight`, it should have `0` value. Can you check in your code the execution of this ternary operator?

Reply:
I'm feeling a bit slow here. Yes. If I set `extraScrollHeight` to 0, then the keyboard space is reset to the appropriate height. But the bug is about when `extraScrollHeight` isn't 0. It's a useful property and it does the right thing when the keyboard is open. When the keyboard is closed, the extra padding it provides should go away and that's what this PR is about.